bundles/org.openhab.binding.astro/src/test/java/org/openhab/binding/astro/internal/calc/SunCalcTest.java

```diff
@@ -0,0 +1,254 @@
+package org.openhab.binding.astro.internal.calc;
```

Comment:
Please add the proper copyright header here

Reply:
Ok.